From 362b2e10bf7a7ba30fe4b7f6fb04fcf7d4297db3 Mon Sep 17 00:00:00 2001
From: harshareddy123 <vardhan1437@gmail.com>
Date: Tue, 18 Apr 2017 15:35:51 +0530
Subject: [PATCH 3/6] jhkjsddsfas

---
 harsha.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/harsha.txt b/harsha.txt
index 29a3343..bf3ba84 100644
--- a/harsha.txt
+++ b/harsha.txt
@@ -3,4 +3,4 @@ Nekante baguna rey.
 on the day 17th feb
 on 21st
 on 10th
-in aslkjn
+in aslkjfsdfsn
-- 
2.11.1.windows.1

